### PR TITLE
Index page added using nuxt3, Vue3 and antd

### PR DESCRIPTION
**Pull request by rmeena840**
Ticket: https://www.pivotaltracker.com/n/projects/2452320/stories/181433099

Work:
Index page added using nuxt3, Vue3 and antd

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -61,6 +61,7 @@ typings/
 
 # Yarn Integrity file
 .yarn-integrity
+.yarn
```
Comment (robert-zaremba) [inline .prettierignore:64]: I thought this was already added ;)

File: pages/index.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+							<a-menu-item key="1"> Algosigner </a-menu-item>
+							<a-menu-item key="2"> Wallet Connect </a-menu-item>
+							<a-menu-item key="3"> My Algo Wallet </a-menu-item>
+						</a-menu>
+					</template>
+					<a-button>
+						Select your wallet
+						<DownOutlined />
+					</a-button>
+				</a-dropdown>
```
Comment (robert-zaremba) [inline pages/index.vue]: please move the wallet select / connect to a separate component (eg `/components/wallet.vue`)

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Let's move wallet select to a separate file / component.

File: .prettierignore (hunk 1 above)
Comment (rmeena840) [reply .prettierignore:64]: No It was not.

File: pages/index.vue (hunk 2 above)
Comment (rmeena840) [reply pages/index.vue]: Done

**Pull request merged**